Documentation/parameters.par

```diff
@@ -149,7 +149,9 @@ xdmfWriterBackend = 'posix' ! (optional) The backend used in fault, wavefield,
 EnergyOutput = 1 ! Computation of energy, written in csv file
 EnergyTerminalOutput = 1 ! Write energy to standard output
 EnergyOutputInterval = 0.05
-ComputeVolumeEnergiesEveryOutput = 4 ! Compute volume energies only once every ComputeVolumeEnergiesEveryOutput * EnergyOutputInterval 
+ComputeVolumeEnergiesEveryOutput = 4 ! Compute volume energies only once every ComputeVolumeEnergiesEveryOutput * EnergyOutputInterval
+
+LoopStatisticsNetcdfOutput = 0 ! Writes detailed loop statistics. Warning: Produces terabytes of data!
```

Comment:
Considering the warning: when does it produce terabytes of data? On which parameters do the statistics depend?

Reply:
I'd like to address this later, when adding documentation. However, for most large runs it will lead to huge amounts of data. Depends linearly on the number of total timesteps (=cluster updates). So scales with number of elements, time step size and also number of MPI ranks.